lexos/processors/prepare/scrubber.py

```diff
@@ -428,6 +294,79 @@ def get_all_punctuation_map() -> Dict[int, type(None)]:
     return punctuation_map
 
 
+def scrub_select_apos(text: str) -> str:
+    """Scrubs all non-word-internal apostrophes from a text.
+
+    :param text: The string to be scrubbed of external apostrophes.
+    :return: The text string, now with only internal apostrophes.
+    """
+
+    # If apos preceded by a non-word character: (?<=[^\w])'
+    # OR apos followed by a non-word character: |'(?=[^\w])
+
+    # Using " " to represent non-word whitespace, "w" to represent a word
+    #     character, and "***" to represent any sequence of any characters,
+    #     this pattern will match:
+    # 1) ***w' *** because the apostrophe is followed by a non-word
+    # 2) *** 'w*** because the apostrophe follows a non-word
+    # 3) *** ' *** because the apos. follows AND is followed by non-words
+    # This will also work for apos. next to other punctuation and multiple
+    #     apos. next to each other, because they count as non-word
+
+    pattern = re.compile(r"(?<=[^\w])'|'(?=[^\w])", re.UNICODE)
```

Comment:
since our definition of words are "groups of none space characters", don't you think uses `\s` is more appropriate?
```regex
(?<=[\s])'|'(?=[\s])
```